src/pipecat/services/ai_services.py

```diff
@@ -235,6 +236,7 @@ async def process_frame(self, frame: Frame, direction: FrameDirection):
                 await self.push_frame(frame, direction)
         elif isinstance(frame, TTSSpeakFrame):
             await self._push_tts_frames(frame.text)
+            await self.flush_audio()
```

Comment:
Same here.

```
    async def process_frame(self, frame: Frame, direction: FrameDirection):
        super().process_frame(frame, direction)
        if isinstance(frame, TTSSpeakFrame):
            await self.flush_audio()
```

Reply:
This should be `await  super().process_frame(frame, direction)`, right?

(An `await` in front of the call to the super class's method.)

Reply:
Yes, sorry.